src/styles/blocks/nav-menu.scss

```diff
@@ -0,0 +1,73 @@
+.nav-menu {
+
+  background-color: $menu-background;
+  &__content {
+    height: 100vh;
+    padding: $header-top-padding 0;
+  }
+
+  &__top {
+    @include top-actions;
+    margin-bottom: 44px;
+
+  }
+
+  &__list {
+    margin: 0 0 44px;
+    padding: 0;
+
+    list-style: none;
+  }
+
+  &__item {
+    &:not(:last-child) {
+      margin-bottom: 24px;
+    }
+  }
+
+  &__link {
+    display: inline-block;
+    padding-bottom: 8px;
+    border-bottom: 1px solid transparent;
+
+    font-weight: 600;
+    font-size: 22px;
+    line-height: 22px;
+    color: $menu-item-color;
+    text-decoration: none;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+
+    transition: border-color $transition linear;
```

Comment:
as i remember, its better to add transition also on hover too (not only in this part of your code)

Reply:
I think not, if I add a transition on hover too it will work to one side